SDLSecurity/SDLSecurityManager.m

```diff
@@ -88,7 +89,7 @@ - (nullable NSData *)decryptData:(NSData *)data withError:(NSError * _Nullable _
 }
 
 + (NSSet<NSString *> *)availableMakes {
-    return [NSSet setWithArray:@[@"Example"]];
+    return [SDLPrivateSecurityConstants.class availableMakes];
```

Comment:
```suggestion
    return SDLPrivateSecurityConstants.availableMakes;
```

Once you make the above change, this should be possible.

Reply:
Fixed.